Bio/SeqUtils/MeltingTemp.py

```diff
@@ -468,7 +473,9 @@ def _check(seq, method):
         )
     if method == "Tm_NN":
         baseset = ("A", "C", "G", "T", "I")
-    seq = "".join([base for base in seq if base in baseset])
+
+    if not all(base in baseset for base in seq):
+        raise ValueError("The input sequence is not valid for " + method)
```

Comment:
f strings are generally preferred to string concatenation, so something like:
```
raise ValueError(f"The input sequence is not valid for {method}")
```
would be every so slightly cleaner

Reply:
Sure - I find this marginal for adding a string variable, but in general prefer f-strings.

Reply:
I also stumbled across this one, but as I'm not sure if we have some sort of f string policy in Biopython and in this example the simple string concatenation has +/- length and readability than the respective f string, I didn't flag this.

Reply:
It’s not important, though f strings have slowly become the de facto standard in Python so I thought I’d at least mention it. Probably the only practical benefit in this case is avoiding that trailing space. Those have an unfortunate tendency to disappear while making changes to code